src/isSymbol.js

```diff
@@ -0,0 +1,32 @@
+import { type } from 'ramda';
+import curry1 from 'ramda/src/internal/_curry1';
+
+/**
+ * Checks if input value is a Symbol.
+ *
+ * @func isSymbol
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.22.0|v2.22.0}
+ * @category Type
+ * @sig * -> Boolean
+ * @param {*} val The value to test
+ * @return {boolean}
+ * @see {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol|MDN Symbol}
+ * @example
+ *
+ * RA.isSymbol(Symbol('1')); //=> true
+ * RA.isSymbol(Symbol(1)); //=> true
+ * RA.isSymbol('string'); //=> false
+ * RA.isSymbol(undefined); //=> false
+ * RA.isSymbol(null); //=> false
+ */
+const isSymbol = curry1(val => {
```

Comment:
The function is returning `true` when `val` is `{}` or `[]` or `new Error()` and I believe it should not return `true`. I did not understand the reason, I guess is something do to with `curry1`. Do you have any suggestions?

Reply:
The problem is caused by bug in your code `typeof val === 'object' || val.constructor.name === 'Symbol'`. This should be joined by `&&`. But this code I suggested to remove anyway in another comment, so no big deal. When this matching statement is removed everything should be working as expected

Reply:
You are right! Thanks.